### PR TITLE
Updated PHP requirements

### PR DESCRIPTION
**Pull request by n3omaster**
When trying to required this package on a PHP 8 environment it returns an error.

### CODE REVIEW

**Review by nomaxus — APPROVED** [review]

**Comment by kapitonoval** [comment]
"lastguest/murmurhash": "^2.0.0" - is not use php8

**Comment by bobbygrossmann** [comment]
Hi i notice murmurhash 2.1.1 supportes php 8

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "bitwasp/bitcoin",
+  "name": "n3omaster/bitcoin-php",
```
Comment (bobbygrossmann) [inline composer.json:2]: This is incorrect?

**Review by bobbygrossmann — CHANGES_REQUESTED** [review]

**Comment by BafS** [comment]
If it can help we maintain a fork with PHP8.0+ compatibility: https://github.com/ProtonMail/bitcoin-php

**Pull request closed without merging**